tests/basic.rs

```diff
@@ -5,9 +5,10 @@ fn test_cat_blocking() {
     use std::io::Write as _;
 
     let mut pty = pty_process::blocking::Pty::new().unwrap();
+    let pts = pty.pts().unwrap();
     pty.resize(pty_process::Size::new(24, 80)).unwrap();
     let mut child = pty_process::blocking::Command::new("cat")
-        .spawn(&pty.pts().unwrap())
+        .spawn(&pts)
         .unwrap();
```

Comment:
Similar to setting nonblocking, other operations also seem to be disallowed prior to obtaining a pts.